.coderabbit.yaml

```
@@ -0,0 +1,64 @@
+    - "State management patterns"
+
+  # Enable detailed analysis
+  enable_code_suggestions: true
+  enable_security_scan: true  
+  enable_performance_analysis: true
+  enable_accessibility_check: true
+
+  # Custom prompts for spiritual content app
+  custom_instructions: |
+    This is a React Native spiritual companion app built with Expo and TypeScript.
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)